src/cascadia/TerminalSettingsModel/ActionArgs.h

```diff
@@ -1526,6 +1527,56 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         }
     };
 
+    struct FocusPaneArgs : public FocusPaneArgsT<FocusPaneArgs>
+    {
+        FocusPaneArgs() = default;
+        FocusPaneArgs(uint32_t id) :
+            _Id{ id } {};
+        WINRT_PROPERTY(uint32_t, Id);
+        static constexpr std::string_view IdKey{ "id" };
+
+    public:
+        hstring GenerateName() const;
+
+        bool Equals(const IActionArgs& other)
+        {
+            auto otherAsUs = other.try_as<FocusPaneArgs>();
+            if (otherAsUs)
+            {
+                return otherAsUs->_Id == _Id;
+            }
+            return false;

```

Comment:
This can be shortened to `otherAsUs && otherAsUs->_Id == _Id`.

Reply:
This one is keeping with the actionargs style -- eventually we will automatically generate these, and having the existing data be uniform helps us validate that :smile: